ravenframework/Optimizers/acquisitionFunctions/AcquisitionFunction.py

```diff
@@ -191,8 +192,8 @@ def testGradients(self, bayesianOptimizer, nSamples):
     funGrad = nd.Gradient(evalMethod, step=0.0001, order=4)
     diffVector = np.empty(nSamples)
     limits = np.array(self._bounds)
-    sampler = LHS(xlimits=limits, criterion='cm', random_state=42)
-    initSamples = sampler(nSamples)
+    initSamples = lhs(limits.shape[0], samples=nSamples, criterion='cm', random_state=42)
+    initSamples = limits[:,0] + (limits[:,1] - limits[:,0]) * initSamples
```

Comment:
I suggest to make a util function for these lines since they appears multiple times and in different modules.

Reply:
We can fix that in new PR; as there are too many things going on here.